### PR TITLE
add django 5.0 to CI/test matrix and project classifiers

### PR DESCRIPTION
**Pull request by charlesobrien**
Adding Django 5.0 to CI/test matrix and bumping some other Django minor versions

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -14,10 +14,14 @@ jobs:
         strategy:
             matrix:
                 python-version: [ 3.8, 3.9, "3.10", "3.11", "pypy3.9" ]
-                django: [ 32, 41, 42]
+                django: [ 32, 41, 42, 50]
                 exclude:
                     - python-version: "3.11"
                       django: 32
+                    - python-version: 3.8
```
Comment (BradleyKirton) [inline .github/workflows/test.yaml:21]: Should the pypy3.9 also be added to the exclusion list for django 5?

[https://docs.djangoproject.com/en/5.0/releases/5.0/#python-compatibility](https://docs.djangoproject.com/en/5.0/releases/5.0/#python-compatibility)

Comment (charlesobrien) [reply .github/workflows/test.yaml]: @BradleyKirton Nice catch; had 3.9 in exclusions but missed pypy3.9. Added now!

**Comment by charlesobrien** [comment]
> please check the build failures

@auvipy Build should be passing now. The primary issue was pytest 8 was failing (all prior builds were running against pytest 7 and the pytest 8 release in Jan 2024 had breaking changes). I had to apply a <8 version constraint and the build would pass.

**Review by browniebroke — COMMENTED** [review]
I'm not a maintainer but may I also suggest to update the classifiers: 

https://github.com/celery/django-celery-results/blob/cb9448976bc51865922319c91b6c2218859ba80d/setup.py#L44-L48

**Comment by browniebroke** [comment]
> I'm not a maintainer but may I also suggest to update the classifiers:
> 
> https://github.com/celery/django-celery-results/blob/cb9448976bc51865922319c91b6c2218859ba80d/setup.py#L44-L48

Opened a PR against your fork to add it https://github.com/charlesobrien/django-celery-results/pull/1

**Comment by charlesobrien** [comment]
> > I'm not a maintainer but may I also suggest to update the classifiers:
> > https://github.com/celery/django-celery-results/blob/cb9448976bc51865922319c91b6c2218859ba80d/setup.py#L44-L48
> 
> Opened a PR against your fork to add it [charlesobrien#1](https://github.com/charlesobrien/django-celery-results/pull/1)

Thanks; I've merged that!

**Comment by charlesobrien** [comment]
@auvipy Following up. Anything else needed for merge?

**Review by auvipy — APPROVED** [review]

**Pull request merged**